maven-plugin/src/main/java/io/confluent/kafka/schemaregistry/maven/DownloadSchemaRegistryMojo.java

```diff
@@ -51,33 +49,48 @@
   @Parameter(required = true)
   List<String> subjectPatterns = new ArrayList<>();
 
+  @Parameter(required = false)
+  List<String> versions = new ArrayList<>();
+
   @Parameter(required = true)
   File outputDirectory;
 
-  Map<String, ParsedSchema> downloadSchemas(Collection<String> subjects)
+  Map<String, ParsedSchema> downloadSchemas(List<String> subjects, List<String> versionsToDownload)
```

Comment:
We need a check that if `versionsToDownload` is non-empty, then the size matches that of `subjects`, otherwise we should throw a `MojoExecutionException` with a nice message